Tasks/PublishToAzureServiceBusV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -3,11 +3,13 @@
   "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=870237)",
   "loc.description": "Sends a message to azure service bus using a service connection (no agent required).",
   "loc.instanceNameFormat": "Publish to Azure Service Bus",
-  "loc.group.displayName.signingDetails": "Signing Properties",
+  "loc.group.displayName.advancedProperties": "Advanced",
   "loc.input.label.connectedServiceName": "Azure Service Bus service connection",
   "loc.input.help.connectedServiceName": "Select an Azure Service Bus service connection.",
   "loc.input.label.messageBody": "Message body",
   "loc.input.help.messageBody": "Enter the json messageBody.",
+  "loc.input.label.sessionId": "Session Id",
+  "loc.input.help.sessionId": "This is the session id with which you want to post the message. If not given for Session Based Queues, Publishing of message will fail for such Queueus. For Non Session Based Queues, it will not matter.",
```

Comment:
There are spelling mistakes "Queueus", incorrect case "Session Based Queues, Publishing", "for such Queues", "Non Session Based Queues" and its directed towards user with words like you..

Change the complete sentence to "Session id with which message is published. For session based queues, publishing fails if value not specified and for non session based queues, value is ignored."

Reply:
This sentence is factually incorrect , so will not change it for now . Will discuss with you.

Reply:
Also i dont think word "you" should not be used as i can see multiple such usages in task markdown. But taking the change to move forward.